firmware/baseband/proc_btlerx.hpp

```diff
@@ -42,48 +43,1931 @@ class BTLERxProcessor : public BasebandProcessor {
     void on_message(const Message* const message) override;
 
    private:
+#define SAMPLE_PER_SYMBOL 1
+#define LEN_DEMOD_BUF_ACCESS 32
+#define DEFAULT_ACCESS_ADDR 0x8E89BED6
+#define NUM_ACCESS_ADDR_BYTE 4
+
+    std::array<std::string, 16> ADV_PDU_TYPE_STR{
+        "ADV_IND",
+        "ADV_DIRECT_IND",
+        "ADV_NONCONN_IND",
+        "SCAN_REQ",
+        "SCAN_RSP",
+        "CONNECT_REQ",
+        "ADV_SCAN_IND",
+        "RESERVED0",
+        "RESERVED1",
+        "RESERVED2",
+        "RESERVED3",
+        "RESERVED4",
+        "RESERVED5",
+        "RESERVED6",
+        "RESERVED7",
+        "RESERVED8"};
+
+    typedef enum {
+        ADV_IND = 0,
+        ADV_DIRECT_IND = 1,
+        ADV_NONCONN_IND = 2,
+        SCAN_REQ = 3,
+        SCAN_RSP = 4,
+        CONNECT_REQ = 5,
+        ADV_SCAN_IND = 6,
+        RESERVED0 = 7,
+        RESERVED1 = 8,
+        RESERVED2 = 9,
+        RESERVED3 = 10,
+        RESERVED4 = 11,
+        RESERVED5 = 12,
+        RESERVED6 = 13,
+        RESERVED7 = 14,
+        RESERVED8 = 15
+    } ADV_PDU_TYPE;
+
+    uint8_t macAddress[6];
+    int checksumReceived = 0;
+
+    typedef struct
```

Comment:
In C++ don't typedef structs, just declare them.

```cpp
struct my_struct {
...
};
```

Reply:
Done